core/src/main/java/org/mule/PropertyScope.java

```diff
@@ -10,25 +10,14 @@
 import java.util.Comparator;
 
 /**
- * A PropertyScope is used to associate a message property with a lifetime.  A scope may be very 
- * brief such as {@link #INVOCATION} which only lasts until a service has been invoked or longer 
- * running such as {@link #SESSION}.
+ * A PropertyScope is used to associate a message property with a lifetime.
  */
 public final class PropertyScope implements Serializable
```

Comment:
If session and invocation scopes are removed, does not make sense to just remove this class and directly use getInboundProperties/getOutboundProperties(name) instead of getProperty(name, scope)?